js/SisyphusSheep.js

```diff
@@ -1140,7 +1140,7 @@ var SisyphusSheepGame = function(){
 			backgroundColor: 0x90a4ae
 		};
 
-		renderer = PIXI.autoDetectRenderer(rendererOptions);
+		var renderer = PIXI.autoDetectRenderer(rendererOptions);
```

Comment:
Why does this break your game? And why does it work without the "var"? And why does your lgtm thing give this as a warning, yet when I do something about it, it breaks your game?

Reply:
renderer needs to be global variable to be used by the library